Admin/AbstractAdmin.php

```diff
@@ -1493,7 +1505,15 @@ public function getLabel()
      */
     public function setPersistFilters($persist)
     {
-        $this->persistFilters = $persist;
+        $this->filterPersister = $persist;
+    }
+
+    /**
+     * @param FilterPersisterInterface $filterPersister
+     */
+    public function setFilterPersister($filterPersister)
```

Comment:
Please use type hinting here, and make it optional so that `null` can be passed.